### DISCUSSION TITLE
search shortcut

### DISCUSSION THREAD

**Discussion by honoklevi** [valinet/ExplorerPatcher#3730] [Ideas]
if they can't bring back the old search bar, then add an option that creates a shortcut on the taskbar, that opens up, a search bar that looks like the 10's, (a shortcut like for chrome etc.) and you can hide the search icon+ maybe somehow make the search bar (not the icon) an option too that does the same.

**Comment by milestprower92** [comment]
if you want the Windows 10 search box on the taskbar (not search menu) use the reimplemented taskbar
